src/OSPSuite.Core/Domain/ISimulation.cs

```diff
@@ -9,5 +10,7 @@ public interface ISimulation : ILazyLoadable, IAnalysable, IUsesObservedData, IM
       IEnumerable<CurveChart> Charts { get; }
 
       OutputMappings OutputMappings { get; set; }
+
+      DataRepository ResultsDataRepository { get; set; }
```

Comment:
Ok let's keep this. Let's make sure we explain that it can be null for some simulations
In fact, I feel that for the population, we should throw an exception saying that this should never be called 
@Yuri05 Thoughts?

Reply:
For Core I have added a comment in the code, then in the solutions we can actually throw the esception.